### PR TITLE
feat: Re-use notes rich editor as new content composer - EXO-72612_EXO-72145 - Meeds-io/MIPs#128

### PR DESCRIPTION
**Pull request by hakermi**
Re-use notes rich editor as new content composer

### CODE REVIEW

File: content-webapp/src/main/webapp/news-activity-composer-app/main.js [hunk 1]
```diff
@@ -30,6 +30,7 @@ const lang = typeof eXo !== 'undefined' ? eXo.env.portal.language : 'en';
 
 // should expose the locale resources as REST API
 const urls = [
+  `${eXo.env.portal.context}/${eXo.env.portal.rest}/i18n/bundle/locale.portlet.notes.notesPortlet-${lang}.json`,
```
Comment (azayati) [inline content-webapp/src/main/webapp/news-activity-composer-app/main.js:33]: does this needed for common notes labels ?

File: content-webapp/src/main/webapp/schedule-news-drawer/components/ExoScheduleNewsDrawer.vue [hunk 2]
```diff
@@ -21,7 +21,7 @@
 <template>
   <div>
     <v-overlay
-      z-index="1000"
+      z-index="3000"
```
Comment (azayati) [inline content-webapp/src/main/webapp/schedule-news-drawer/components/ExoScheduleNewsDrawer.vue:24]: do this change have a impact on old news composer ?

**Review by azayati — APPROVED** [review]
LGTM

File: content-webapp/src/main/webapp/schedule-news-drawer/components/ExoScheduleNewsDrawer.vue (hunk 2 above)
Comment (hakermi) [reply content-webapp/src/main/webapp/schedule-news-drawer/components/ExoScheduleNewsDrawer.vue:24]: Nope

File: content-webapp/src/main/webapp/news-activity-composer-app/main.js (hunk 1 above)
Comment (hakermi) [reply content-webapp/src/main/webapp/news-activity-composer-app/main.js:33]: yes needed to not duplicate the labels of the editor
for the translation components for example

**Pull request merged**